research/conditional/utils/model_utils.py

```diff
@@ -245,6 +248,46 @@ def get_expert_choice_args(args):
     }
 
 
+def get_expert_choice_with_parallel_ff_args(args):
+    assert args.granularity_expert_config, "Must use granularity config"
```

Comment:
This assert should be removed in this form, because we are going to deprecate parser argument `--granularity_expert_config`: pull request https://github.com/llm-random/llm-random/pull/169

Reply:
Done.